irs_reader/settings.py

```diff
@@ -1,68 +1,159 @@
-import sys
 import os
+
+import environ
+
 from .dir_utils import mkdir_p
 
-IRS_READER_ROOT = os.path.abspath(os.path.dirname(__file__))
+env = environ.Env()
+
+
+IRS_READER_ROOT = env(
+    "IRS_READER_ROOT", default=os.path.abspath(os.path.dirname(__file__))
+)
 
-# This is the URL to amazon's bucket, could use another synced to it
-IRS_XML_HTTP_BASE = "https://s3.amazonaws.com/irs-form-990"
+# This is the URL to Giving Tuesday's bucket, could use another synced to it
+IRS_XML_HTTP_BASE = env(
```

Comment:
@asuozzo, to your point about making the XML paths configurable, i made the path to the xml files settable environmental variable.